django/contrib/gis/gdal/datasource.py

```diff
@@ -87,8 +87,10 @@ def __init__(self, ds_input, ds_driver=False, write=False, encoding='utf-8'):
 
     def __del__(self):
         "Destroys this DataStructure object."
-        if self._ptr and capi:
```

Comment:
` and capi` was added in 8f334e55be99ae271d82f9b961a2dc6258487d9c to prevent AttributeError, was it unneeded?